helm/plain-node/values.yml

```diff
@@ -0,0 +1,50 @@
+replicaCount: 1
+
+name: "plain-node-x"
+
+testnetName: "testnet"
+
+image:
+  repository: codaprotocol/coda-daemon
```

Comment:
Is this the correct repository? [codaprotocol/coda-daemon](https://hub.docker.com/r/codaprotocol/coda-daemon) seems to be 6 months old, whereas [minaprotocol/mina-daemon](https://hub.docker.com/r/minaprotocol/mina-daemon) is quite fresh